benchmark/scripts/Benchmark_Driver

```diff
@@ -346,17 +350,40 @@ def positive_int(value):
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Swift benchmarks driver')
-    subparsers = parser.add_subparsers()
-
-    submit_parser = subparsers.add_parser(
-        'submit',
-        help='run benchmarks and submit results to LNT')
-    submit_parser.add_argument(
+    parser = argparse.ArgumentParser(
+        epilog='Example: ./Benchmark_Driver run -i 5 -f Array'
+    )
+    subparsers = parser.add_subparsers(
+        title='Swift benchmark driver commands',
+        help='See COMMAND -h for additional arguments', metavar='<command>')
+
+    parent_parser = argparse.ArgumentParser(add_help=False)
+    benchmarks_group = parent_parser.add_mutually_exclusive_group()
+    benchmarks_group.add_argument(
+        'benchmark',
```

Comment:
Benchmarks are positional arguments - just a list of names at the end of the line:

```
mondo$ $BUILD/bin/Benchmark_Driver run -o O MonteCarloE AngryPhonebook
1,AngryPhonebook,1,13177,13177,13177,0,13177,11046912
124,MonteCarloE,1,50568,50568,50568,0,50568,18423808

Totals,2,63745,63745,63745,0,63745,29470720
```